snapshot/bundlestore/browse_store.go

```diff
@@ -0,0 +1,76 @@
+package bundlestore
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/scootdev/scoot/os/temp"
+)
+
+// Wraps another store and, when reading and when bundle name contains a slash, returns interpreted data.
```

Comment:
OHHHH, I think I see. I was confusing what this is.

Yeah, I think this will be easier in the future with SnapshotDB, but this is fine for now...